gp-templates/glossary-view.php

```diff
@@ -68,7 +68,7 @@
 
 				<form action="<?php echo esc_url( gp_url_join( $url, '-new' ) ); ?>" method="post">
 					<dl>
-						<dt><label for="new_glossary_entry_term"><?php esc_html( 'Original term:', 'glossary entry', 'glotpress' ); ?></label></dt>
+						<dt><label for="new_glossary_entry_term"><?php echo esc_html( _ex( 'Original term:', 'glossary entry', 'glotpress' ) ); ?></label></dt>
```

Comment:
`_ex()` should only be `_x()` in this case as `_ex()` echo's the result instead of returning it ;)